frontend/public/components/nav/section.tsx

```diff
@@ -209,6 +209,7 @@ export const NavSection = connect(navSectionStateToProps)(
             isActive={isActive}
             isExpanded={isOpen}
             onExpand={this.toggle}
+            data-cy="nav"
```

Comment:
This should be `data-test` per [our best practices](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/README.md#migrating-protractor-tests-to-cypress)